frontend/src/redux/Filters/filters.actions.ts

```diff
@@ -0,0 +1,20 @@
+import { createAsyncAction } from 'typesafe-actions';
+import { ISelectedOptionsType } from '../../components/MultiSelect/MultiSelect.type';
+
+export const selectProjectIds = createAsyncAction(
+  'Filters/SELECT_PROJECT_IDS_REQUEST',
+  'Filters/SELECT_PROJECT_IDS_SUCCESS',
+  'Filters/SELECT_PROJECT_IDS_FAILURE',
+)<
+  {
+    selectedProjectIds: ISelectedOptionsType[];
+  },
+  {
```

Comment:
Je trouve cette syntaxe difficile à lire, on peut peut être nommer les différents arguments en expliquant ce qu'ils représentent (et du coup les setter à {} si besoin) ça évitera les commentaires not used qui n'indiquent pas vraiment comment on pourrait s'en servir.

Reply:
💬  Je crois que c'est le but même de createAsyncAction, qui permet de typer le payload de l'action request, success et error.